### ISSUE TITLE
XP Potions

### ISSUE THREAD

**Issue by KCCAsiPri** [DiscordDungeons/Bugs#721]
*Describe the Bug*:
The XP potions that were made for me and Templexx do not work. The potions don't boost XP by 1.25.

*To Reproduce*
No way, except for to see in game how there is no change

**Comment by RainTheVappy** [comment]
Found 13 items
Charisma Potion, Defense Potion, Divine Protection of Water, Even Greater Health Potion, Felix's Potion, Greater Health Potion, Health Potion, Luck Potion, Nova Potion, Potion of Poison, Potion of Wonders, Strength Potion, Undead Potion
What's the potion name?

**Comment by Templexx** [comment]
My potion is "PoT"

**Comment by RainTheVappy** [comment]
It boosts the XPBoost attribute and I guess it won't display on #!attribs

**Comment by KCCAsiPri** [comment]
My potion is "Royal Elixir".
Botu mine and Templexx's are 1.25 boost.

**Comment by FelixArgyle** [comment]
Yeah noticed the same thing, thinking it has to do with xp ring is nulling it out possibly, thought I reported it already tho 🤔

**Comment by KCCAsiPri** [comment]
It should stack, rite?

**Comment by KCCAsiPri** [comment]
Wait then is it possible for me to get a refund for my potion then... if it
serves no purpose in the end it becomes useless. Not what I spent my money
for.

**Comment by PlayTheFallen** [comment]
I'm completely blind...

**Comment by KCCAsiPri** [comment]
It should be a multiplier.
Mine is 1.25x

**Comment by Sven65** [comment]
Checking the code, the order of the xp items are boots, chestplate, helmet, weapon and necklace as a single value, then the rings multiplier, and lastly the potions multiplier. checking the code also revealed that the potions are set to use the "luck" attribute, and not the new ones.

**Comment by KCCAsiPri** [comment]
So would it be possible to alter the potion's attribute-in-effect so that it matches the new "XPBoost" one?

**Comment by Sven65** [comment]
I need to that in the code, but yes.
